File: Fi-Thniti-web(symfony)/src/Entity/Carpooling.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Carpooling
 *
 * @ORM\Table(name="carpooling", indexes={@ORM\Index(name="userid", columns={"userid"})})
 * @ORM\Entity
 */
class Carpooling
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="departure_date", type="date", nullable=false)
     */
    private $departureDate;

    /**
     * @var string
     *
     * @ORM\Column(name="departure_location", type="string", length=50, nullable=false)
     */
    private $departureLocation;

    /**
     * @var string
     *
     * @ORM\Column(name="drop_off_location", type="string", length=50, nullable=false)
     */
    private $dropOffLocation;

    /**
     * @var string
     *
     * @ORM\Column(name="driver_name", type="string", length=20, nullable=false)
     */
    private $driverName;

    /**
     * @var int
     *
     * @ORM\Column(name="phone_number", type="integer", nullable=false)
     */
    private $phoneNumber;

    /**
     * @var int
     *
     * @ORM\Column(name="places_number", type="integer", nullable=false)
     */
    private $placesNumber;

    /**
     * @var string
     *
     * @ORM\Column(name="baggage", type="string", length=3, nullable=false)
     */
    private $baggage;

    /**
     * @var string|null
     *
     * @ORM\Column(name="preference", type="string", length=255, nullable=true, options={"default"="NULL"})
     */
    private $preference = 'NULL';

    /**
     * @var \Users
     *
     * @ORM\ManyToOne(targetEntity="Users")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="userid", referencedColumnName="user_id")
     * })
     */
    private $userid;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getDepartureDate(): ?\DateTimeInterface
    {
        return $this->departureDate;
    }

    public function setDepartureDate(\DateTimeInterface $departureDate): self
    {
        $this->departureDate = $departureDate;

        return $this;
    }

    public function getDepartureLocation(): ?string
    {
        return $this->departureLocation;
    }

    public function setDepartureLocation(string $departureLocation): self
    {
        $this->departureLocation = $departureLocation;

        return $this;
    }

    public function getDropOffLocation(): ?string
    {
        return $this->dropOffLocation;
    }

    public function setDropOffLocation(string $dropOffLocation): self
    {
        $this->dropOffLocation = $dropOffLocation;

        return $this;
    }

    public function getDriverName(): ?string
    {
        return $this->driverName;
    }

    public function setDriverName(string $driverName): self
    {
        $this->driverName = $driverName;

        return $this;
    }

    public function getPhoneNumber(): ?int
    {
        return $this->phoneNumber;
    }

    public function setPhoneNumber(int $phoneNumber): self
    {
        $this->phoneNumber = $phoneNumber;

        return $this;
    }

    public function getPlacesNumber(): ?int
    {
        return $this->placesNumber;
    }

    public function setPlacesNumber(int $placesNumber): self
    {
        $this->placesNumber = $placesNumber;

        return $this;
    }

    public function getBaggage(): ?string
    {
        return $this->baggage;
    }

    public function setBaggage(string $baggage): self
    {
        $this->baggage = $baggage;

        return $this;
    }

    public function getPreference(): ?string
    {
        return $this->preference;
    }

    public function setPreference(?string $preference): self
    {
        $this->preference = $preference;

        return $this;
    }

    public function getUserid(): ?Users
    {
        return $this->userid;
    }

    public function setUserid(?Users $userid): self
    {
        $this->userid = $userid;

        return $this;
    }
    public function __toString(){
        $var = strval($this->id);
        return $var;
    }


}

File: Fi-Thniti-Desktop(javafx)/src/entities/carpooling.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;
import java.util.*;
import java.sql.Date;

/**
 *
 * @author 1h3b
 */

public class carpooling {
    private int carpooling_id;
    private Date departure_date;
    private String departure_location;
    private String drop_off_location;
    private String driver_name;
    private int phone_number;
    private int places_number;
    private String baggage;
    private String preference;
    
    public carpooling(int carpooling_id,Date departure_date,String departure_location,String drop_off_location,String driver_name,int phone_number,int places_number,String baggage,String preference) {
           this.carpooling_id=carpooling_id;
           this.departure_date=departure_date;
           this.departure_location=departure_location;
           this.drop_off_location=drop_off_location;
           this.driver_name=driver_name;
           this.phone_number=phone_number;
           this.places_number=places_number;
           this.baggage=baggage;
           this.preference=preference;
          
}

    public carpooling() {
        //throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
    }

    public int getCarpooling_id() {
        return carpooling_id;
    }

    public void setCarpooling_id(int carpooling_id) {
        this.carpooling_id = carpooling_id;
    }

    public Date getDeparture_date() {
        return departure_date;
    }

    public void setDeparture_date(Date departure_date) {
        this.departure_date = departure_date;
    }

    public String getDeparture_location() {
        return departure_location;
    }

    public void setDeparture_location(String departure_location) {
        this.departure_location = departure_location;
    }

    public String getDrop_off_location() {
        return drop_off_location;
    }

    public void setDrop_off_location(String drop_off_location) {
        this.drop_off_location = drop_off_location;
    }

    public String getDriver_name() {
        return driver_name;
    }

    public void setDriver_name(String driver_name) {
        this.driver_name = driver_name;
    }

    public int getPhone_number() {
        return phone_number;
    }

    public void setPhone_number(int phone_number) {
        this.phone_number = phone_number;
    }

    public int getPlaces_number() {
        return places_number;
    }

    public void setPlaces_number(int places_number) {
        this.places_number = places_number;
    }

    public String getBaggage() {
        return baggage;
    }

    public void setBaggage(String baggage) {
        this.baggage = baggage;
    }

    public String getPreference() {
        return preference;
    }

    public void setPreference(String preference) {
        this.preference = preference;
    }

    @Override
    public String toString() {
        return "carpooling{" + "carpooling_id=" + carpooling_id + ", departure_date=" + departure_date + ", departure_location=" + departure_location + ", drop_off_location=" + drop_off_location + ", driver_name=" + driver_name + ", phone_number=" + phone_number + ", places_number=" + places_number + ", baggage=" + baggage + ", preference=" + preference + '}';
    }

    

   
    
   
    
    
}

